test/call.js

```diff
@@ -19,7 +19,7 @@ describe("eth_call", function() {
   var estimateGasContractAbi;
   var EstimateGasContract;
   var estimateGasInstance;
-  var source = fs.readFileSync(path.join(__dirname, "EstimateGas.sol"), "utf8");
+  var source = fs.readFileSync(path.join(__dirname, "/contracts/gas/EstimateGas.sol"), "utf8");
```

Comment:
shouldn't have slashes in the `path.join` call, as `path.join` is there to figure out the path separators for you

should be: `path.join(__dirname, "contracts", "gas", "EstimateGas.sol")`

Reply:
Can change this but will be fixed in a future PR.

Reply:
Changed.